docs/Getting Started/Debian/Debian Buster Root on ZFS.rst

```diff
@@ -52,6 +52,7 @@ Contributing
 #. Install the tools::
 
     sudo apt install pip3
+    cd openzfs-docs/docs
```

Comment:
I've merged this idea (but in a different way) in 371d3cf08ba235e68b36df3285975b4ca450b7bd.